terraform/variables.tf

```diff
@@ -0,0 +1,64 @@
+variable "environment_name" {
+  type = string
+}
+
+variable "azure_sp_credentials_json" {
+  type    = string
+  default = null
+}
+
+variable "key_vault_name" {
+  type = string
+}
+
+variable "resource_group_name" {
+  type = string
+}
+
+variable "paas_api_url" {
+  default = "https://api.london.cloud.service.gov.uk"
+}
+
+variable "paas_org_name" {
+  type    = string
+  default = "dfe"
+}
+
+variable "paas_space" {
+  type = string
+}
+
+variable "apply_qts_app_name" {
```

Comment:
Can we remove this variable and determine the name in code? Like `apply-for-qts-in-england` + `environment_name`